### ISSUE TITLE
Can DisablePatchGuard be used to load only one driver while rejecting all others?

### ISSUE THREAD

**Issue by lolcocks123** [arielkoren/DisablePatchGuard#3]
So I am using [DIfferentSLIAuto](https://www.techpowerup.com/forums/threads/sli-with-different-cards.158907/) and it works great!
It uses modified drivers (which are unsigned to run non SLI cards in SLI mode).

But with new games the problem is that their anti cheats do not allow games to be run when Windows is in test mode.
Even with full DisablePatchGuard, anti cheats detect that testing mode is on.

So,
**tl;dr, is it possible to load just one driver (nvlddmkm.sys) using DisablePatchGuard while rejecting all others?**
